token/program/src/state.rs

```diff
@@ -287,6 +287,64 @@ fn unpack_coption_u64(src: &[u8; 12]) -> Result<COption<u64>, ProgramError> {
     }
 }
 
+const SPL_TOKEN_ACCOUNT_MINT_OFFSET: usize = 0;
+const SPL_TOKEN_ACCOUNT_OWNER_OFFSET: usize = 32;
+const SPL_TOKEN_ACCOUNT_LENGTH: usize = 165;
+
+/// A trait of the generic token account for unpacking an account's fields
+/// partially and returning their references efficiently.
```

Comment:
```suggestion
/// A trait for token Account structs to enable efficiently unpacking various fields 
/// without unpacking the complete state.
```

Reply:
This could probably use some more wordsmithing.

Reply:
Done